todo/services/task_service.py

```
@@ -111,6 +112,9 @@ def prepare_task_dto(cls, task_model: TaskModel) -> TaskDTO:
         assignee = cls.prepare_user_dto(task_model.assignee) if task_model.assignee else None
         created_by = cls.prepare_user_dto(task_model.createdBy)
         updated_by = cls.prepare_user_dto(task_model.updatedBy) if task_model.updatedBy else None
+        deferred_details = (
+            cls.prepare_deferred_details_dto(task_model.deferredDetails) if task_model.deferredDetails else None
```

Pylint (3.3.7)

[convention] Line too long (112/100)

(C0301)